internal/rpc/core/routes.go

```diff
@@ -28,6 +28,7 @@ func (env *Environment) GetRoutes() RoutesMap {
 		"block_results":        rpc.NewRPCFunc(env.BlockResults, "height", true),
 		"commit":               rpc.NewRPCFunc(env.Commit, "height", true),
 		"check_tx":             rpc.NewRPCFunc(env.CheckTx, "tx", true),
+		"remove_tx":            rpc.NewRPCFunc(env.RemoveTx, "txkey,removeFromCache", false),
```

Comment:
we should try to avoid putting this on the public interface. Any one could potentially constantly remove all txs. Possibly group into the unsafe section?

Reply:
I don't necessarily see how that changes anything, as the unsafe section are just as available to users as the safe section

Reply:
the unsafe section makes the operator aware they are turning it on. This change will expose functionality that could alter the nodes behaviour, that was my reasoning for suggesting unsafe. 

I could crawl the network, find nodes and get a list of unconfirmed txs and remove all of them. If its a node that is a sentry or a validator it would clear the mempool. If we are fine exposing such behaviour, it should be documented somehow that this can cause problems.